packages/terra-table/docs/guides/SectionWithinSection.md

```diff
@@ -0,0 +1,144 @@
+# Terra Table - Sections with Subsections
+
+With the inclusion of sections in the table, there are two recommended patterns for created nested collapsible sections. First is collapsible sections and static subsections, and the second is static sections and collapsible subsections. The guidance from UX is to not have both collapsible sections and collapsible subsections in a table.
+
+## State Management
+As section and subsection have the same API, we'll be walking through the expectation of a collapsible section in only one pattern.
+
+First defaulting our state to an empty array in the constructor. 
+```jsx
+  constructor(props) {
+    super(props);
+
+    this.state = { collapsedKeys: [] };
+  }
+```
+Next creating an event handler callback method to pass to the section's "onSelect" prop. The "onSelect" callback will return the metaData prop passed to each section.
+```jsx
+  handleSectionSelection(event, metaData) {
+
+  }
+```
+As a precaution we can prevent default on the event, in case the table has an ancestor with a listener. This also prevents the browser from auto page scrolling when we are intending to make a selection with the space bar.
+```jsx
+  handleSectionSelection(event, metaData) {
+    event.preventDefault();
+  }
+```
+Terra Table comes with additional helpers to manage state, in this case we want to determine if the selection has collapsed or opened the section using the section key in our state. So we use the utility method "updatedMulitSelectedKeys", which returns an array of the keys following the addition or removing of the key passed. We then set this as our state.
```

Comment:
Might be helpful to add a link to the doc details of `updatedMulitSelectedKeys`

Reply:
Updated doc for updatedMultiSelectedKeys.